### PR TITLE
PRINE-136 add monitoring endpoints

### PR DESCRIPTION
**Pull request by M16**

### CODE REVIEW

File: functions/src/api/controller/monitoring.controller.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  @ApiOperation({
+    summary: 'Calls all gate control devices and checks their status',
+  })
+  @ApiResponse({
+    status: 200,
+    description: 'Successful pong back',
+    type: MonitoringResponseDto,
+  })
+  @ApiResponse({ status: 404, description: 'Not Found' })
+  async monitoring(): Promise<string> {
```
Comment (hirsch88) [inline functions/src/api/controller/monitoring.controller.ts:27]: should the content of this method not be in a service?
it looks like business logic :-)

**Review by hirsch88 — CHANGES_REQUESTED** [review]

File: functions/src/api/controller/monitoring.controller.ts (hunk 1 above)
Comment (hirsch88) [reply functions/src/api/controller/monitoring.controller.ts:27]: Maybe also try to use async/await instead of .then

Comment (prine) [reply functions/src/api/controller/monitoring.controller.ts:24]: I have updated it and its now an own service with asynch/await.

**Pull request merged**